doc/semitrans.xml

```diff
@@ -251,3 +251,83 @@ Transformation( [ 7, 8, 8, 7, 8, 8, 8, 7, 8, 8 ] )]]></Example>
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DigraphOfActionOnPairs">
+  <ManSection>
+    <Attr Name = "DigraphOfActionOnPairs" Arg = "coll" Label = "for ever" />
+    <Oper Name = "DigraphOfActionOnPairs" Arg = "coll[, n]" Label = "never" />
+    <Returns>A digraph.</Returns>
+    <Description>
+      If <A>coll</A> is a transformation collection and <A>n</A> is a
+      non-negative integer, then <C>DigraphOfActionOnPairs(<A>coll</A>,
+        <A>n</A>)</C> returns a digraph representing the <Ref Oper="OnSets"
+        BookName="ref"/> action of the transformations in <A>coll</A> on the
+      pairs of points in <C>[1 ..  <A>n</A>]</C>.  If the optional argument
+      <A>n</A> is not specified, then by default the value of
+      <C>DegreeOfTransformationCollection(<A>coll</A>)</C> will be chosen for
+      <A>n</A>.  <P/>
+
+      The <C>DigraphOfActionOnPairs</C> has <M><A>n</A> + {n \choose 2}</M>
+      vertices: the vertices <C>[1 .. <A>n</A>]</C> correspond to the points in
+      <C>[1 .. <A>n</A>]</C>, and the remaining vertices correspond to the pairs
+      of points in <C>[1 .. <A>n</A>]</C>. This correspondence is stored in
+      <Ref Oper="DigraphVertexLabels" BookName="Digraphs" />.
+      <P/>
+
+      The edges of <C>DigraphOfActionOnPairs</C> are defined as follows,
+      depending on whether or not <A>coll</A> is a semigroup. If <A>coll</A> is
+      a semigroup, define <C>gens</C> to be
+      <C>GeneratorsOfSemigroup(<A>coll</A>)</C>; otherwise define <C>gens =
+        <A>coll</A></C>.
```

Comment:
Isn't this a bit complicated? Wouldn't it make more sense to just define this as an operation on a collection of transformations? This would be at the cost of recalculating this it is needed more than once, and also a bit more typing, but would be more "pure" (I guess?)